third-party/genuine-fantasy-press/forgotten-secrets/invocations.xml

```diff
@@ -1338,11 +1344,346 @@
 		<prerequisite>Wild Huntsman patron, Pact of the Blade feature</prerequisite>
 		<requirements>ID_GFP_COFSA_ARCHETYPE_OTHERWORLDLY_PATRON_WILD_HUNTSMAN, ID_WOTC_PHB_CLASS_FEATURE_WARLOCK_PACT_BOON_PACT_OF_THE_BLADE</requirements>
 		<description>
+			<p class="flavor">Prerequisite: Wild Huntsman patron, Pact of the Blade feature</p>
 			<p>You can now create any simple or martial weapon from blackened cold iron and the bone of ancient beasts using your Pact of the Blade feature. If this weapon is a ranged weapon, you can instantly create ammunition for it in your open hand. Whenever you hit a creature with this weapon while they are the subject of hunter’s mark, the target takes an extra 1d6 cold damage. You can choose to use Strength for attack and damage rolls with this pact weapon, even if it is ranged.</p>
 		</description>
 		<sheet>
 			<description>You can now create any simple or martial weapon using your Pact of the Blade feature. If this weapon is a ranged weapon, you can instantly create ammunition for it in your open hand. Whenever you hit a creature with this weapon while they are the subject of hunter’s mark, the target takes an extra 1d6 cold damage. You can choose to use Strength for attack and damage rolls with this pact weapon, even if it is ranged.</description>
 		</sheet>
 	</element>
 
-</elements>
\ No newline at end of file
+	<element name="Awakened Insight" type="Class Feature" source="The Compendium of Forgotten Secrets" id="ID_GFP_COFSA_CLASS_FEATURE_AWAKENED_INSIGHT">
```

Comment:
might as well include the `_ELDRITCH_INVOCATION` after `_CLASS_FEATURE` in the ID for the new invocations, specially in such a large source the chance for duplication is higher.